### ISSUE TITLE
Stencil builder cache seems to ignore changes to global scripts

### ISSUE THREAD

**Issue by dimi-nk** [ionic-team/stencil#716]
**Stencil version:**

```
 @stencil/core@0.7.18
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**

Changes in the global scripts are not picked up by the stencil compiler when serving locally (not sure about prod build).

**Expected behavior:**

Any change to the global script is immediately picked up

**Steps to reproduce:**

As per stenciljs guide, I've created 2 files `src/global/index.ts` & `src/global/highcharts.ts`. Then I've added the following property to my stencil config `globalScript: 'src/global/index.ts'`. The first time it worked fine.

Then I did changes to the `highcharts.ts` file, but nothing happened. I looked into the generated files under `www/build/xxx/xxx.global.js` and the code seems to be the one from before. Changes to `highcharts.ts` are not picked up.

If I disable the cache in `stencil.config.js` it works as expected, but cannot even re-enable it (still keeps using the old contents).

**Related code:**

```ts
// index.ts
import highcharts from './highcharts';

declare var Context: any;

Context.highcharts = highcharts;
```

```ts
// highcharts.ts
import Highcharts from 'highcharts';

const highcharts = Highcharts;

export default (function () {
    return highcharts;
})();
```

```js
// stencil.config.js
exports.config = {
    ...
    globalScript : 'src/global/index.ts'
};
```

**Other information:**

**Comment by dimi-nk** [comment]
Might be related to #575 but cannot be sure

**Comment by NorthMcCormick** [comment]
For the record I am running into the exact same issue but I can't figure out how to even get stencil to recognize the new code. I've re-cloned my directories, cleared NPM's cache, deleted the www dir and re-built with every possible setting.

Does anyone know a way to get around this until it's fixed?

**Comment by paulmaclean** [comment]
@NorthMcCormick , did you try adding enableCache: false to the stencil config?

**Comment by NorthMcCormick** [comment]
@paulmaclean I had just come across that late last night hahaha. That does work.

For the record when re-enabled still uses the old cache. (I think this behavior is described in another related ticket, too)

**Comment by dimi-nk** [comment]
From this ticket 😂 
> If I disable the cache in stencil.config.js it works as expected, but cannot even re-enable it (still keeps using the old contents).

**Comment by dimi-nk** [comment]
Do we have any updates for this? We've been missing build caching for some time now..

**Comment by dimi-nk** [comment]
Just upgraded to `0.9.7` and the issue looks resolved.

**Comment by brendensoares** [comment]
I can confirm that this seems to be resolved in `0.9.11`. Works as expected even with caching enabled.
